From 923e39a43c5338347cdc8faff8742f7210d791df Mon Sep 17 00:00:00 2001
From: Jan Chaloupka <jchaloup@redhat.com>
Date: Mon, 30 Jul 2018 14:04:20 +0200
Subject: [PATCH] Fix formatting

---
 rounder_example_test.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rounder_example_test.go b/rounder_example_test.go
index 803c1d7..29626cb 100644
--- a/rounder_example_test.go
+++ b/rounder_example_test.go
@@ -31,7 +31,7 @@ func ExampleRounder() {
 		{"RoundHalfEven", inf.RoundHalfEven}, {"RoundExact", inf.RoundExact},
 	}
 
-	fmt.Println("The results of new(inf.Dec).Round(x, s, inf.RoundXXX):\n")
+	fmt.Println("The results of new(inf.Dec).Round(x, s, inf.RoundXXX):")
 	w := tabwriter.NewWriter(os.Stdout, 0, 0, 1, ' ', tabwriter.AlignRight)
 	fmt.Fprint(w, "x\ts\t|\t")
 	for _, r := range rounders {
-- 
2.7.5

